src/Voron/StorageEnvironment.cs

```diff
@@ -158,6 +158,9 @@ public StorageEnvironment(StorageEnvironmentOptions options)
 
                 if (isNew == false && _options.ManualSyncing == false)
                     SuggestSyncDataFile(); // let's suggest syncing data file after the recovery
+
+                // Ensure we are always have the prefetcher available.
+                var prefetcher = GlobalPrefetchingBehavior.GlobalPrefetcher.Value;
```

Comment:
```suggestion
                GC.KeepAlive(GlobalPrefetchingBehavior.GlobalPrefetcher.Value);
```

We use this commonly, and it is guaranteed that the compiler won't elide this

Reply:
Ok, for readability purposes it makes a lot of sense. But, there is no way thr compiler could elide it, because of the static side effects. If that would happen, there are a few invariants violated (major regression).